docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.7'
+
+
+services:
+  nostr-proxy:
+    image: nostr-proxy
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "127.0.0.1:3003:3000"
```

Comment:
Can you set the exposed PORT to 3000? It's confusing to have another port here I think